x-pack/plugins/cases/public/components/create/form.tsx

```diff
@@ -70,6 +73,10 @@ export const CreateCaseFormFields: React.FC<CreateCaseFormFieldsProps> = React.m
     const { isSubmitting } = useFormContext();
     const { isSyncAlertsEnabled } = useCasesFeatures();
 
+    const { owner } = useCasesContext();
+    const availableOwners = useAvailableCasesOwners();
+    const canShowCaseTypeSelection = !owner.length && availableOwners.length;
```

Comment:
nit: `canShowCaseTypeSelection` -> `canShowCasePluginSelection`

Reply:
Let's keep this consistent with the result of the discussion below. If we go with `Solution` then maybe we name it `canShowCaseSolutionSelection`?

Reply:
fixed